changelogs/changelog.yaml

```diff
@@ -2,139 +2,137 @@ ancestor: null
-    release_date: "2023-07-26"
+    - make_action_operation.yml
+    - support_parent_directory.yaml
+    - update_tests.yaml
+    release_date: '2023-07-26'
   4.1.0:
     changes:
       doc_changes:
-        - Update badges to refer to the proper GitHub repository.
+      - Update badges to refer to the proper GitHub repository.
       minor_changes:
-        - Add tag support to persist task.
+      - Add tag support to persist task.
     fragments:
-      - implement_tag.yaml
-      - update_badge.yaml
-    release_date: "2023-11-29"
+    - implement_tag.yaml
+    - update_badge.yaml
+    release_date: '2023-11-29'
   5.0.0:
     changes:
       major_changes:
-        - Bumping `requires_ansible` to `>=2.14.0`, since previous ansible-core versions
-          are EoL now.
-      release_summary:
-        Starting from this release, the minimum `ansible-core` version
+      - Bumping `requires_ansible` to `>=2.14.0`, since previous ansible-core versions
+        are EoL now.
+      release_summary: Starting from this release, the minimum `ansible-core` version
         this collection requires is `2.14.0`. The last known version compatible with
         ansible-core<2.14 is `v4.1.0`.
     fragments:
-      - trivial_lint.yaml
-    release_date: "2023-12-01"
+    - trivial_lint.yaml
+    release_date: '2023-12-01'
   6.0.0:
     changes:
       doc_changes:
-        - Eliminate the redundant installation section.
-        - Update README.
-        - Update the installation steps with correct url.
+      - Eliminate the redundant installation section.
+      - Update README.
+      - Update the installation steps with correct url.
       major_changes:
-        - Bumping `requires_ansible` to `>=2.15.0`, since previous ansible-core versions
-          are EoL now.
-      release_summary:
-        With this release, the minimum required version of `ansible-core`
+      - Bumping `requires_ansible` to `>=2.15.0`, since previous ansible-core versions
+        are EoL now.
+      release_summary: With this release, the minimum required version of `ansible-core`
         for this collection is `2.15.0`. The last version known to be compatible with
         `ansible-core` versions below `2.15` is v5.0.0.
     fragments:
-      - bump_215.yaml
-      - remove_redundant.yaml
-      - update_readme.yaml
-    release_date: "2024-06-12"
+    - bump_215.yaml
+    - remove_redundant.yaml
+    - update_readme.yaml
+    release_date: '2024-06-12'
   7.0.0:
     changes:
       major_changes:
-        - Restructured the network.base collection by converting supported operations
-          into separate roles.
+      - Restructured the network.base collection by converting supported operations
+        into separate roles.
     fragments:
-      - fix_ci.yaml
-      - restructure_operations_into_roles.yaml
-    release_date: "2025-01-17"
+    - fix_ci.yaml
+    - restructure_operations_into_roles.yaml
+    release_date: '2025-01-17'
+  8.0.0:
```

Comment:
add a major release note and release summary explaining why this change is required, and revert old changes or check with pre-commit locally